src/ts/localization/de.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
it's a good idea and also taught about it. I think mario disagreed on this part. also as a personal opinion : it would make things much more complicated if we want to *merge* the languages together, and also getting a translation would require a lot of *checks* if the property even exists ...etc. at this point I don't really see the benefit of doing so other than it being *better* for the eye

Reply:
:+1: then we should agree on an interface with correct names, ..etc. before I do the change. 
should we add it to specification issue ?